resources/views/ItemList/Components/Filter/ItemFilterImage.twig

```diff
@@ -4,7 +4,7 @@
             <div class="form-check" v-for="value in facet.values" :key="value.id">
                 <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading">
                 <label :for="'option-' + value.id" class="form-check-label" rel="nofollow">
-                    <img :src="value.imageUrl" @error="handleImageError($event)" width="80px"/>
+                    <img :src="value.imageUrl" @error="handleImageError($event)" :width="value.imageUrl ? '80px' : ''"/>
```

Comment:
How would this look like? Can you add a screenshot of before and after, so we can decide which one we want to use.

Reply:
Before empty space would be left after the filter is selected for the image, even when there is no image:
https://snipboard.io/8YVMkm.jpg
After:
https://snipboard.io/3FE1IH.jpg

Reply:
Thank you, we will leave it as it is.